locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "scalar interval": "स्केलर इंटरवल",
+  "milliseconds": "मिलीसेकंड",
+  "silence": "मौन",
+  "down": "नीचे",
+  "up": "ऊपर",
+  "pitch": "पिच",
+  "Silence block cannot be removed.": "मौन अवरोध हटाया नहीं जा सकता",
+  "You can restore deleted blocks from the trash with the Restore From Trash button.": "आप हटाए गए ब्लॉकों को कचरे से पुनर्स्थापित कर सकते हैं 'कचरे से पुनर्स्थापित करें' बटन द्वारा।",
+  "true": "सत्य",
+  "false": "असत्य",
+  "Cartesian": "कार्टेसियन",
+  "polar": "पोलर",
+  "Cartesian/Polar": "कार्टेसिय",
+  "none": "कोई नहीं",
+  "treble": "ट्रेबल",
+  "grand staff": "ग्रैंड स्टाफ",
+  "mezzo-soprano": "मेज़्ज़ो-सोप्रानो",
+  "alto": "आल्टो",
+  "tenor": "टेनर",
+  "bass": "बास",
+  "on2": "ऑन2",
+  "off": "ऑफ",
+  "Not a number": "एक संख्या नहीं",
+  "Octave value must be between 1 and 8.": "ऑक्टेव मान 1 से 8 के बीच होना चाहिए।",
+  "Numbers can have at most 10 digits.": "संख्याओं में अधिकतम 10 अंक हो सकते हैं।",
+  "box": "बक्सा",
+  "Consider breaking this stack into parts.": "इस स्टैक को भागों में बाँटने का विचार करें।",
+  "open file": "फ़ाइल खोलें",
+  "text": "पाठ",
+  "store in box": "बॉक्स में स्टोर करें",
+  "box1": "बॉक्स1",
+  "box2": "बॉक्स2",
+  "store in": "में भंडारित करें",
+  "name": "नाम",
+  "value": "मान",
+  "Forever loop detected inside a note value block. Unexpected things may happen.": "नोट वैल्यू ब्लॉक के अंदर एक अनंत लूप पाया गया है। अप्रत्याशित चीज़ें हो सकती हैं।",
+  "There is no block selected.": "कोई ब्लॉक चयन नहीं किया गया है।",
+  "avatar": "अवतार",
+  "sample": "सैम्पल",
+  "mouse": "माउस",
+  "brown rat": "भूरा चूहा",
+  "mole": "मौज",
+  "chipmunk": "चिपमंक",
+  "red squirrel": "लाल गिलहरी",
+  "guinea pig": "गिनीपिग",
+  "capybara": "कैपीबारा",
+  "coypu": "कोयपू",
+  "black rat": "काला चूहा",
+  "grey squirrel": "स्लेटी गिलहरी",
+  "flying squirrel": "उड़ती गिलहरी",
+  "bat": "बैट",
+  "start drum": "ड्रम चालू करें",
+  "Not a valid pitch name": "एक वैध पिच नाम नहीं",
+  "You must select a file.": "आपको एक फ़ाइल का चयन करना होगा।",
+  "Playback is ready.": "प्लेबैक तैयार है।",
+  "Grid": "ग्रिड",
+  "effect": "प्रभाव",
+  "sargam": "सरगम",
+  "custom pitch": "कस्टम पिच",
+  "accidental": "अक्सीडेंटल",
+  "pitch converter": "पिच कनवर्टर",
+  "double sharp": "डबल शार्प",
+  "sharp": "शार्प",
+  "natural": "नैचुरल",
+  "flat": "फ्लैट",
+  "double flat": "डबल फ्लैट",
+  "unison": "एकस्वरता",
+  "major": "मेजर",
+  "ionian": "आयनियन",
+  "minor": "माइनर",
+  "aeolian": "एोलियन",
+  "Duplicate": "डुप्लीकेट",
+  "Extract": "निकालें",
+  "Move to trash": "कूड़े में डालें",
+  "Close": "बंद करें",
+  "Save stack": "स्टैक को सेव करें",
+  "Help": "मदद",
+  "In order to copy a sample, you must reload the widget, import the sample again, and export it.": "सैंपल कॉपी करने के लिए, आपको विजेट को फिर से लोड करना होगा, सैंपल को पुनः आयात करना होगा, और फिर इसे निर्यात करना होगा।",
+  "You have chosen key for your pitch preview.": "आपने अपनी पिच पूर्वावलोकन के लिए कुंजी चुनी है।",

```

This is fine. But open to hearing your thoughts on using चाबी?
```suggestion
  "You have chosen key for your pitch preview.": "आपने अपनी पिच पूर्वावलोकन के लिए चाबी चुनी है।",

```